scripts/west_commands/runners/nrfjprog.py

```diff
@@ -73,9 +73,6 @@ def do_add_parser(cls, parser):
         parser.add_argument('--softreset', required=False,
                             action='store_true',
                             help='use reset instead of pinreset')
-        parser.add_argument('--snr', required=False,
-                            help="""Serial number of board to use.
-                            '*' matches one or more characters/digits.""")
```

Comment:
Let's keep this for compatibility, but feel free to adjust the help text accordingly.